python/arcticdb/arctic.py

```diff
@@ -26,7 +26,7 @@ class Arctic:
 
     _LIBRARY_ADAPTERS = [S3LibraryAdapter, LMDBLibraryAdapter, AzureLibraryAdapter]
 
-    def __init__(self, uri: str, encoding_version: EncodingVersion = EncodingVersion.V1):
+    def __init__(self, uri: str, encoding_version=EncodingVersion.V1, check_storage_access=True):
```

Comment:
It's a shame we introduced `encoding_version` here - some sort of `ArcticOptions` object would probably have been better. Now for a user to change `check_storage_access` they also need to check the default for `encoding_version`, whereas they could have created an `ArcticOptions` instance with the defaults and tweaked individual attributes.

Do you think it's too late to change? I would be very surprised if anyone is trying the new encoding version...

Reply:
We can definitely change it, I don't think anyone will be using it at the moment.

Reply:
I'll make `check_storage_access` kw-only so less impacted if we remove `encoding_version`